models/articles.models.js

```diff
@@ -36,16 +36,26 @@ exports.updateVotesOfArticleByID = (articleId, newVote) => {
     });
 };
 
-exports.fetchArticles = (sort_by = "created_at", order = "DESC") => {
+exports.fetchArticles = (sort_by = "created_at", order = "DESC", topic) => {
   const validSortBy = ["created_at"];
   const validOrder = ["asc", "ASC", "desc", "DESC"];
+  const queryValues = [];
 
   let queryStr = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, 
   COUNT(comments.article_id) AS comment_count 
   FROM articles 
   LEFT JOIN comments ON articles.article_id = comments.article_id 
   GROUP BY articles.article_id`;
 
+  if (topic) {
```

Comment:
You could make this way dryer by separating out the GROUP BY from the original string, then doing your conditional logic of if(topic) to add the topic query if necessary, then adding the GROUP BY back on regardless of whether a topic has been added or not!